src/main/java/io/github/railroad/settings/ui/themes/Theme.java

```diff
@@ -3,6 +3,9 @@
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
+/**
+ * Information about a theme.
+ */
```

Comment:
This is a completely useless javadoc. If it doesn't say anything useful then it shouldn't exist.